src/python/doc/barycenter_user.rst

```diff
@@ -0,0 +1,51 @@
+:orphan:
+
+.. To get rid of WARNING: document isn't included in any toctree
+
+Wasserstein distance user manual
```

Comment:
copy-paste?

Reply:
Indeed. Should I keep the header about the toctree btw ?

Reply:
The question about toctree is for @VincentRouvreau , but my guess is that you have to keep some arbitrary text there, so it might as well be text that explains why it is there.

Reply:
We have no toctree element in our documentation. It seems annoying for sphinx as it outputs some warning for each `*.rst`. You can keep `:orphan:` and the comment to fix the warning.